artemis-server/src/main/java/org/apache/activemq/artemis/core/config/WildcardConfiguration.java

```diff
@@ -178,6 +180,22 @@ public String convert(final String input, final WildcardConfiguration target) {
       }
    }
 
+   /**
+    * Detect whether the input {@code String} contains the "any words" or "single word" character and is not escaped.
+    */
+   public boolean isWild(String input) {
+      return input == null ? false : ((input.contains(getAnyWordsString()) || input.contains(getSingleWordString())) && !isEscaped(input));
```

Comment:
Should `isWild` return true when it include one `any words` character escaped and one `any words` character not escaped?

Reply:
That is a good point. Let me have a think about this...

Reply:
This shouldn't be a problem anymore. :+1: